tree/treeplayer/src/TTreePlayer.cxx

```diff
@@ -2461,43 +2461,43 @@ Long64_t TTreePlayer::Scan(const char *varexp, const char *selection,
 
    if (opt.Contains("lenmax=")) {
       int start = opt.Index("lenmax=");
-      int numpos = start + strlen("lenmax=");
```

Comment:
This function could also use the shortening of the name (and in general any function where there is more than one use of the function).

Reply:
Changed as suggested